src/Caller.h

```diff
@@ -58,6 +58,11 @@
 * 
 */
 class Caller {
+
+  std::istream& getDataInStream(std::ifstream& fileStream);
+  bool loadAndNormalizeData(std::istream &dataStream, XMLInterface& xmlInterface, SetHandler& setHandler, Scores& allScores);
+  void calcAndOutputResult(Scores& allScores, XMLInterface& xmlInterface, time_t& procStart, clock_t& procStartClock, double& diff);
+
```

Comment:
If the preceding methods are actually intended to be private as opposed to protected, then explicitly mark them as such -- don't rely on the reader to know that private is the default. (And private fields should be declared at the bottom of class definitions anyway.) But it seems to me that they ought to be declared as protected.

Reply:
I'm not sure myself if they should be private or protected. Since other parts of the code having similar functions are protected I guess they maybe should be protected as well.